src/main/kotlin/com/mirf/features/dicomimage/data/DicomData.kt

```diff
@@ -0,0 +1,99 @@
+package com.mirf.features.dicomimage.data
+
+import com.mirf.core.array.BooleanArray2D
+
+import com.mirf.core.data.medimage.ImagingData
+import com.pixelmed.dicom.DicomOutputStream
+import com.pixelmed.dicom.TagFromName
+import java.awt.image.BufferedImage
+import java.io.File
+import java.io.FileOutputStream
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+import javax.imageio.stream.FileImageInputStream
+
+/**
+ * Realize ImageData on dicom
```

Comment:
This javadoc is unclear. I cannot understand from it what this class is doing. Make it more specific.